HW12/SQLAlchemy/linguist.py

```
@@ -0,0 +1,147 @@
+def deck_update(deck_id, name):
+    update_deck = s.query(Decker).filter_by(id=deck_id).first()
+    update_deck.name = name
+    s.commit()
+    return Decker
+
+
+def deck_delete_by_id(deck_id):
+    delete_deck = s.query(Decker).filter_by(id=deck_id).first()
+    s.delete(delete_deck)
+    s.commit()
+    return True if not s.query(Decker).filter_by(id=deck_id).all() else False
```

Ruff (0.8.2)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)